README.md

```diff
@@ -1 +1,142 @@
 # servo-rancher
+Optune servo driver for Rancher (v1.6)
+
+This driver supports an application deployed via Rancher as a Stack.
+
+A Rancher Stack may contain multiple containers, each of which may individually be modified by the adjust driver.
```

Comment:
is 'containers' the correct Rancher object? Shouldn't this be 'services?

Reply:
Its project (environment in the UI) -> stack -> service -> container

But yes, in this case it should be service.